web_external/js/views/body/StudyResultsView.js

```diff
@@ -436,16 +448,29 @@ isic.views.StudyResultsLocalFeaturesView = isic.View.extend({
 
 // View for an image
 isic.views.StudyResultsImageView = isic.View.extend({
-    initialize: function (settings) {
-        this.listenTo(this.model, 'change', this.render);
+    setVisible: function (visible) {
+        if (visible) {
+            this.$el.removeClass('hidden');
+
+            if (this.imageViewerWidget) {
```

Comment:
When would we be calling `.setVisible(true)` on a `StudyResultsImageView` that hasn't yet be rendered? Or is this just to deal with a race condition during initialization?